lesson03.ipynb

```diff
@@ -0,0 +1,443 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.        trace = pm.sample(1000, chains=2, return_inferencedata=True)
</pre><p>Still need a likelihood to get something to fit.  Same as with linear regression.  You can have f=X*beta, still need the pm.Normal("lik", mu=f, ...) for anything to happen</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bwengals/etc/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Ah I guess I did a bad job of reading these docs
https://docs.pymc.io/en/v3/pymc-examples/examples/gaussian_processes/GP-Latent.html